private-templates-service/src/app.ts

```diff
@@ -1,21 +1,52 @@
-import express from 'express';
-import path from 'path';
-import mongoose from 'mongoose';
-import passport from './config/passport';
+import express from "express";
+import path from "path";
+import mongoose from "mongoose";
+import passport from "./config/passport";
+import bodyParser from "body-parser";
+import mongo from "connect-mongo";
+import session from "express-session";
+import bluebird from "bluebird";
+import logger from "./util/logger";
 
 // import controllers
-import templateRouter from './controllers/template';
+import templateRouter from "./controllers/template";
 
 const app = express();
 
-// connect to temp mongoDB
+// connect to temp mongoDB (will be replaced by adapter)
+const MongoStore = mongo(session);
+mongoose.Promise = bluebird;
+
+mongoose
+  .connect("mongodb://localhost/card", {
+    useNewUrlParser: true,
+    useCreateIndex: true,
+    useUnifiedTopology: true
+  })
+  .then(() => {})
+  .catch(err => {
+    logger.error("Mongodb connection error " + err);
+  });
 
 // Express configuration
 app.use(express.static(path.join(__dirname, "./client/build")));
 app.use(passport.initialize());
 app.use(passport.session());
+app.use(
+  session({
+    resave: true,
+    saveUninitialized: true,
+    secret: "test",
+    store: new MongoStore({
+      url: "mongodb://localhost/adaptivecard",
```

Comment:
You don't have to, but I would add a todo to move these parameters (specifically the URL) out to a config file somewhere so we dont forget later

Reply:
Done, thanks!